web/src/components/MemoContent/CodeBlock.tsx

```diff
@@ -28,24 +33,35 @@ const CodeBlock: React.FC<Props> = ({ language, content }: Props) => {
     // Skip error and use default highlighted code.
   }
 
+  if (formatedLanguage === MERMAID_LANGUAGE) {
+    languageSpecificClasses = "mermaid";
+  }
```

Comment:
Let's return the renderer directly as same as https://github.com/usememos/memos/pull/2971/files#diff-e7203928564407fe22b273bea681a744e690877dc88daf9cac949fc5f09e011cR23

Reply:
@boojack Thank you for catching it up 👍 . Updated to return renderer directly.

Unfortunatelly, the link you've provided doesn't seem to point anywhere specific in the code.